bedrock/press/templates/press/speaker-request.html

```diff
@@ -1,59 +1,64 @@
-{% extends "base-resp.html" %}
+{# This Source Code Form is subject to the terms of the Mozilla Public
+  # License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "base-protocol-mozilla.html" %}
 
 {% block page_title %}{{_('Speaker Request Form')}}{% endblock %}
-{% block body_id %}press_speaker_request{% endblock %}
-{% block body_class %}sand{% endblock %}
 
 {% block page_css %}
-  {{ css_bundle('press') }}
+{{ css_bundle('press') }}
 {% endblock %}
 
 {% block content %}
-
-<main role="main" id="main-content">
-  <header class="main-header">
-    <h1 class="title-shadow-box">{{_('Speaker request form')}}</h1>
-  </header><!-- /header -->
-
-  <section class="container">
-  {% if not form_success %}
-    <h2 class="secondary-title">{{ _('Please complete and submit the form below to request a speaker from Mozilla for your upcoming event.') }}</h2>
-  {% endif %}
-    <div class="main-column">
-    {% if not form_success %}
-      <p class="callout">
-      {% trans url='https://bugzilla.mozilla.org/form.dev-engagement-event' %}
+<main class="mzp-l-content">
+  <h1>{{_('Speaker request form')}}</h1>
+  <section class="mzp-c-emphasis-box">
+    <div class="mzp-l-content">
+      {% if not form_success %}
+      <h2>
+        {{ _('Please complete and submit the form below to request a speaker from Mozilla for your upcoming event.') }}
+      </h2>
+      {% endif %}
+      {% if not form_success %}
+      <p class="highlight">
```

Comment:
If feels like semantically, this could be an emphasis box, rather than the whole form?